assets/scripts/streets/StreetName.jsx

```diff
@@ -116,6 +103,7 @@ export default class StreetName extends React.Component {
 }
 
 StreetName.propTypes = {
+  id: React.PropTypes.string,
   allowEditing: React.PropTypes.bool,
   street: React.PropTypes.any,
   handleResize: React.PropTypes.func
```

Comment:
Oh - I missed this comment yesterday, but this is exactly what I was thinking. We were on the same page all along!